### PR TITLE
Added feater for credits expiration with celery

### PR DESCRIPTION
**Pull request by bashaashma**
# Description

Added feature for expiration of credits using celery

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Trello Ticket ID

https://trello.com/c/pTc0nnpQ/1139-add-functionality-for-expiry-of-promotional-credits-expire-after-6-months

## How Can This Be Tested?

Start the application. Run celery worker and beat as specified in the readme. A users credits should expire at midnight if their credits expiration is on that date.  If needed, under tasks/__init__.py change the time of the celery scheduler (beat) on line 66 to test and check for expired credits in a shorter period of time i.e. crontab().

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MutegekiHenry — APPROVED** [review]
Great work on this @bashaashma

File: app/tasks/__init__.py [hunk 1]
```diff
@@ -46,6 +58,62 @@ def send_async_email(email,
...
+
+    db = SQLAlchemy()
+
+    db.create_all()
+
+    print("Executing task : Checking credit expiry...")
+    credit_assignment_schema = CreditAssignmentSchema(many = True)
+
+    todays_datetime = datetime(datetime.today().year, datetime.today().month, datetime.today().day)
+    #         SELECT c.user_id, c.amount,  u.expiry_date
```
Comment (e-ian) [inline app/tasks/__init__.py:80]: I believe these commented out code can now be removed

**Review by e-ian — APPROVED** [review]
Just a small nit, otherwise LGTM

File: app/tasks/__init__.py (hunk 1 above)
Comment (bashaashma) [reply app/tasks/__init__.py:80]: Sounds good. Thanks

**Review by e-ian — APPROVED** [review]
LGTM

**Pull request merged**